.travis.yml

```diff
@@ -17,12 +22,12 @@ jobs:
     - stage: validate
       name: Lint Commits
       script:
-        - npm install @commitlint/travis-cli
+        - yarn global add @commitlint/travis-cli
```

Comment:
why global?

Reply:
copy/pasta. i'll just dev install it on travis